Pipfile

```diff
@@ -16,6 +16,7 @@ tensorboardX = "==2.0.*"
 requests = "~=2.22"
 transformers = "~=2.3"
 radam = {git = "https://github.com/LiyuanLucasLiu/RAdam"}
+sentencepiece = ">=0.1.83"
```

Comment:
you need to add the dependency to setup.py too, also you should probably set the dependency to be "<0.2.0" as well.

Reply:
fixed